### DISCUSSION TITLE
Question about "bounties"

### DISCUSSION THREAD

**Discussion by Chillenburg** [Aelto/W3_RandomEncounters_Tweaks#83] [Q&A]
So I started my first one and proceeded to 1 of the 5 markers only to have geralt comment "on to the next one" or "theres my target" the bubble disappeared and treated me as though I did it even though I didn't find anything or any monsters. I think the 3rd one I went to had a couple monsters there and the rest all acted just like the 1st - When I got to the last location it auto completed and I received a bounty level of 1.

Is this normal? I'm playing enhanced edition with the enhanced edition RER preset selected.

**Comment by Aelto** [comment]
This is definitely not intended. Since you are using EE, it cannot be the empty bestiary restricting the creatures you can see. I also checked and there is nothing that could potentially block the spawn of the monsters for that part of the code, could it be that you are using a `Death threshold distance` that is lower then 50 meters? Such a value would cause the bounty to be recycled before you even reach the creature.

**Reply by Chillenburg** [reply]
This death threshold distance modifier, would this be in RER or w3ee? Just got to work so I cant test anything for another 12 hours. If it helps, I'm playing w3ee redux. All settings in RER, and w3ee redux have not been adjusted past the default values.

**Reply by Aelto** [reply]
Okay, the setting i talked about it from RER. But since you say you didn't go past the default settings it may not be that, it' may still be worth to check that slider once you're back home.

In the mean time i will check if anything is wrong. If you start new bounties and still get the issue, please tell me 👍

**Reply by Aelto** [reply]
I have multiple reports about bounties starting with a list full of Unknown creatures like so:
```
- 1 Unknown
- 1 Unknown
- 1 Unknown
```

It is confirmed something broke on that end, i will look into it!

**Reply by Aelto** [reply]
The bug is now fixed, you can install the [new release](https://github.com/Aelto/tw3-random-encounters-reworked/releases/tag/v2.7-pre-release_4) (you can use the update script in the mod folder to save time).

Thanks for the report and sorry for the inconvenience. If you don't want to go back to the bounty master to start a new bounty, use the `rerchallenge` console command to start a new one.

**Reply by Chillenburg** [reply]
Yes, it did come up as all "unknowns"  wow, thanks so much for the quick responses and a fast update. 

Edit: Coming up as unknown makes sense considering the bounty master didn't know what was killing his men so I thought that was normal.

**Reply by Aelto** [reply]
Yeah i just realised that "Unknown" may not be a good text for what's supposedly a bug 😁 Good to know it was your issue, then once you're back home today the issue will be resolved. Note that your current bounty won't work though, you'll need to start a new one.

**Comment by Chillenburg** [comment]
Sounds good, if I encounter anything anything else I'll create another
topic in the Q&A section.

**Comment by Chillenburg** [comment]
I ran the update in powershell, I assumed it wint into the base folder like
the auto install script you have - it said everything went fine and I did
the rerchallenge to reset. It said I have 4 targets of -Unknown and the
first bubble I went to disappeared. Checking now to make sure the update
actually did happen.

On Sun, Jan 2, 2022 at 1:15 PM Chillenb ***@***.***> wrote:

> Sounds good, if I encounter anything anything else I'll create another
> topic in the Q&A section.
>

Edit: I just manually applied the update, I can see the names of what im hunting so I think i'm good xD Thanks!
